### ISSUE TITLE
Links in messages

### ISSUE THREAD

**Issue by stefan-mihaila** [LN-Juggernaut/juggernaut-desktop#50]
Juggernaut should detect links in messages and present a clickable option to visit the site. Well, I guess there are some security implications also when any stranger on the lightning network can send you links, so maybe should we handle that somehow, either by disabling links from untrusted contacts or adding an option in the future settings menu?

What about link previews?

**Comment by johncantrell97** [comment]
Yeah, I think parsing out links and presenting them as actual clickable links is a good idea.

Link previews are kind of a problem potentially because without a third party server the message passes through to generate the preview it means the client will have to automatically ping the url and generate the preview.  People might not to want to automatically 'click' links like that.

Another alternative would be that the sender actually generates the preview and sends it as part of their message but this might be problematic because there's a message size limit we'd likely run into.
